### ISSUE TITLE
Tasks without labels

### ISSUE THREAD

**Issue by ronilaukkarinen** [mwakerman/kanbanist#103]
First of all, Kanbanist idea is awesome! I have lots of tasks without labels and I cannot see them in Kanbanist? Do I need to label them first in Todoist to add them to Kanbanist list, or how should I do this?

**Comment by mwakerman** [comment]
Thanks @ronilaukkarinen - do you have Todoist Premium? If so you should be able to create your "Lists" in Todoist and it will add them as labels automatically in Todoist.

**Comment by ronilaukkarinen** [comment]
Yes I do have todoist Premium. For some reason not all tasks from all projects are visible on Kanbanist even though they are there. What do you mean I should create my "Lists" in Todoist? I have projects, labels and tasks already existing there and I want to see them on Kanbanist. Some of them do, but for example from one project I don't see any tasks no matter what I do even though that project has like 20 tasks I would like to sort out on Kanbanist. That Project is shared for other users, could that be the culprit?

**Comment by mwakerman** [comment]
Apologies, I meant to say if you make your lists in Kanban.ist then they'll appear as labels in Todoist.

Tasks with shared labels won't show up in Kanban.ist currently. I've made the change that should allow them to show up which you can preview here: https://deploy-preview-96--kanbanist.netlify.app See #82 for more information about shared labels.

**Comment by ronilaukkarinen** [comment]
Oh. I already did some lists in Kanbanist and they are in sync with Todoist, that's not a problem.

Also, shared labels are not a problem currently.

I try to be as clear as possible.

What is an issue for me is that tasks WITHOUT any labels (thus not on any list, right?) are not showing in Kanbanist at all. Do I need to label them in Todoist? Can't you just have "Unlabeled" list on Kanban to store them? or something?

**Comment by mwakerman** [comment]
Oh, you just need to open your Backlog.

**Comment by ronilaukkarinen** [comment]
Oh, that's what Backlog means! Thanks!

While I'm here, is there a way to Kanbanist to remember my filters when I close the browser and open it again?

**Comment by mwakerman** [comment]
Filters are stored in the URL so the way to save a "board" is to bookmark the URL.
